api/db/changelog/db.changelog-118-update-demographics.xml

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog
+    xmlns="http://www.liquibase.org/xml/ns/dbchangelog/1.9"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog/1.9
+                    http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-1.9.xsd">
+  <changeSet author="srubenst" id="db.changelog-116-update-demographics">
```

Comment:
It might be worth adding a comment here, and/or in the PR description, referencing https://precisionmedicineinitiative.atlassian.net/browse/RW-4312 (thanks for filing!). I also marked that ticket as blocked by RW-4293 for clarity.

Reply:
Threw a comment in there!